src/config-types.ts

```diff
@@ -43,6 +45,7 @@ export type Options = {
         | 'optimize';
       src_include: RegExp;
       src_exclude: RegExp | null;
+      transformer?: UrlTransformer;
```

Comment:
Please add a comment, it will go in the documentation automatically

Reply:
done